fedimint-core/src/util/mod.rs

```diff
@@ -313,8 +313,29 @@ pub use backon::{
     FibonacciBuilder as FibonacciBackoff,
 };
 
-/// Run the supplied closure `op_fn` up to `max_attempts` times. Wait for the
-/// supplied `Duration` `interval` between attempts
+/// Run the supplied closure `op_fn` until it succeeds. Frequency and number of
+/// retries is determined by the specified strategy.
+///
+/// ```
+/// use std::time::Duration;
+///
+/// use fedimint_core::util::{retry, FibonacciBackoff};
+/// # tokio_test::block_on(async {
```

Comment:
Looks like udeps has issues with doc-tests so we'll need to update `fedimint-core/Cargo.toml`

```
[package.metadata.cargo-udeps.ignore]
development = ["tokio-test"]
```

https://github.com/est31/cargo-udeps?tab=readme-ov-file#ignoring-some-of-the-dependencies

Reply:
Thx for the pointer, saved me a lot of time!